bindings/Modules/src/SofaPython3/SofaBaseTopology/CMakeLists.txt

```diff
@@ -22,5 +22,5 @@ SP3_add_python_module(
     DESTINATION  Sofa
     SOURCES      ${SOURCE_FILES}
     HEADERS      ${HEADER_FILES}
-    DEPENDS      SofaBaseTopology SofaPython3::Plugin
+    DEPENDS      Sofa.Component.Topology.Container.Grid SofaPython3::Plugin
```

Comment:
Should be propagated to [Bindings.ModulesConfig.cmake.in](https://github.com/sofa-framework/SofaPython3/blob/master/bindings/Modules/Bindings.ModulesConfig.cmake.in)